libsolidity/analysis/OverrideChecker.cpp

```diff
@@ -573,7 +573,17 @@ void OverrideChecker::checkOverride(OverrideProxy const& _overriding, OverridePr
 			);
 	}
 
-	if (_super.isFunction())
+	if (_super.isModifier())
+	{
+		if (_overriding.unimplemented() && !_super.unimplemented())
```

Comment:
Can you extract the case for function and only create one error (inserting the "modifier" / "function" string based on `isModifier()`)?